src/KafkaFlow.OpenTelemetry/KafkaFlowActivitySourceHelper.cs

```diff
@@ -0,0 +1,38 @@
+﻿extern alias SemanticConventions;
+
+namespace KafkaFlow.OpenTelemetry
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.Reflection;
+    using Conventions = SemanticConventions::OpenTelemetry.Trace.TraceSemanticConventions;
+
+    internal static class KafkaFlowActivitySourceHelper
```

Comment:
Consider renaming to something like `KafkaFlowActivitySourceAccessor`.
(Similar to HttpContextAccessor that provides access to HttpContext)

Or event just `ActivitySourceAccessor` since we are already in the KafkaFlow context

Reply:
Done, please review https://github.com/Farfetch/kafkaflow/pull/451/commits/f02fe4cbaa15aade949cea7ab3b2199cbd831063